platform/android/MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/location/LocationComponentTest.kt

```diff
@@ -97,6 +97,26 @@ class LocationComponentTest {
     Assert.assertEquals(locationEngineRequest, locationComponent.locationEngineRequest)
   }
 
+  @Test
+  fun activateWithDefaultLocationEngineRequestAndOptionsTest() {
+    locationComponent.activateLocationComponent(context, mockk(), true, locationEngineRequest, locationComponentOptions)
+
+    Assert.assertEquals(locationEngineRequest, locationComponent.locationEngineRequest)
+
+    doReturn(mock(TypedArray::class.java)).`when`(context)
+            .obtainStyledAttributes(R.style.mapbox_LocationComponent, R.styleable.mapbox_LocationComponent)
+
+    val resources = mock(Resources::class.java)
+
+    doReturn(resources).`when`(context).resources
+    doReturn(0f).`when`(resources)
+            .getDimension(R.dimen.mapbox_locationComponentTrackingMultiFingerMoveThreshold)
+    doReturn(0f).`when`(resources)
+            .getDimension(R.dimen.mapbox_locationComponentTrackingMultiFingerMoveThreshold)
+    locationComponent.activateLocationComponent(context, mockk(), true, locationEngineRequest, locationComponentOptions)
```

Comment:
This call duplicates the one above, you can skip it and keep only the assertions.

Reply:
Fixed